yt/frontends/enzo/answer_testing_support.py

```diff
@@ -122,8 +121,8 @@ def get_analytical_solution(self):
         pos, dens, vel, pres, inte = np.loadtxt(self.solution_file, unpack=True)
         exact = {}
         exact["pos"] = pos
-        exact["Density"] = dens
```

Comment:
I'm not sure about these -- here we are explicitly using the on-disk fields, I think to prevent any unit conversions being made implicitly or explicitly.  The fields would become `"enzo", "Density"` etc.  Do you want them to be on-disk fields, or are you OK with the unit conversions being applied?

Reply:
Thanks for the comment! I was also worried about unit conversions and was trying to keep with the `"Enzo"` fields. I had to use `"gas"` in the end because the `Pressure` and `ThermalEnergy` fields aren't `"enzo"` fields in the shock tube tests.

To give context, I have just updated (enzo-project/enzo-dev#190) all of the tests in Enzo to be compatible with yt-4.1. The unit conversions don't cause a problem since everything is in cgs for these tests.

Reply:
If you're happy, I'm happy.